PeopleAndAppleStockPrices/PeopleAndAppleStockPrices/Controllers/StockFiles/StocksDetailViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  StocksDetailViewController.swift
+//  PeopleAndAppleStockPrices
+//
+//  Created by 马文龙 on 9/5/19.
+//  Copyright © 2019 Pursuit. All rights reserved.
+//
+
+import UIKit
+
+class StocksDetailViewController: UIViewController {
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var stockImage: UIImageView!
+    @IBOutlet weak var openLabel: UILabel!
+    @IBOutlet weak var closedLabel: UILabel!
+    @IBOutlet var viewColor: UIView!
+    
+    public var stockSelected: StockModel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updateStock()
+        
+    }
+    private func updateStock() {
+        guard let stockSelected = stockSelected else {fatalError("stock is nil")}
+        dateLabel.text = stockSelected.date
+        openLabel.text = String(format: "$%.2f", stockSelected.open)
+        closedLabel.text = String(format: "$%.2f", stockSelected.close)
+        if stockSelected.open >= stockSelected.close {
```

Comment:
This is a good implementation. However the check to see if the stock increased or decreased should really be a function in the model. Do the same thing but make it in the model with the stored properties of open and close.